WordPress/src/main/java/org/wordpress/android/datasets/ReaderCommentTable.java

```diff
@@ -294,6 +294,16 @@ public static boolean isCommentLikedByCurrentUser(long blogId, long postId, long
                 args);
     }
 
+    public static boolean commentExists(long blogId, long postId, long commentId) {
+        String[] args = {Long.toString(blogId),
+                Long.toString(postId),
+                Long.toString(commentId)};
+
+        return SqlUtils.longForQuery(ReaderDatabase.getReadableDb(),
```

Comment:
This is fine, just wanted to toss in a version which uses the approach I've used elsewhere in this package:
```
return SqlUtils.boolForQuery(ReaderDatabase.getReadableDb(),
                "SELECT 1 FROM tbl_comments WHERE blog_id=? AND post_id=? AND comment_id=?",
                args);
```